c/src/ml-api-remote-service.c

```diff
@@ -154,6 +155,120 @@ _mlrs_get_service_type (gchar * service_str)
   return service_type;
 }
 
+/**
+ * @brief Get ml remote service activation type.
+ */
+static gboolean
+_mlrs_parse_activate (gchar * activate)
+{
+  gboolean ret = TRUE;
+
+  if (activate && g_ascii_strcasecmp (activate, "false") == 0) {
```

Comment:
Maybe, It can be changed, g_ascii_strcasecmp return 0 if activate is NULL.
if (g_ascii_strcasecmp (activate, "false") == 0)

Reply:
Right. Changed.